WordPressAuthenticator/Sources/Private/WPAuthenticator-Swift.h

```diff
@@ -1,8 +0,0 @@
-// Import this header instead of <WordPressAuthenticator/WordPressAuthenticator-Swift.h>
-// This allows the pod to be built as a static or dynamic framework
-// See https://github.com/CocoaPods/CocoaPods/issues/7594
-#if __has_include("WordPressAuthenticator-Swift.h")
-    #import "WordPressAuthenticator-Swift.h"
-#else
-    #import <WordPressAuthenticator/WordPressAuthenticator-Swift.h>
-#endif
```

Comment:
This was only useful for the CocoaPods setup. I noticed it grepping for CocoaPods and removed it. There were only two usages anyway.